src/provider.ts

```diff
@@ -253,7 +253,7 @@ export class AnkrProvider {
     private async send<TReply>(method: string, params: any): Promise<TReply> {
         const request = {method, params, id: (this._nextId++), jsonrpc: "2.0"};
         const response = await axios.post<JsonRPCPayload>(this.url, JSON.stringify(request), this.requestConfig);
-        return AnkrProvider.getResult(response.data) as TReply
+        return await AnkrProvider.getResult(response.data) as TReply;
```

Comment:
Hi @mdqst, thank you for the contribution! Could you please elaborate a bit on this line? To the best of my understanding, getResult is a synchronous one, so no need in the await.

PS: I'm not a js/ts expert, so I can be wrong here.

Reply:
Hey @mdqst, thanks for the explanation.

>  using await proactively prevents potential issues.

Sorry, I'd disagree here. 
- it's synchronous now, so the `await` on it would be just misleading, and it decreases readability
- it's for local parsing of already received JSON, so the possibility of refactoring it to something async is close to zero.

So I'd request to remove it from the PR. Please also see the comment from @memekas  regarding types.